Modules/constants_generated.h

```diff
@@ -192,6 +194,7 @@ add_int(OPT_URI);
 add_int(OPT_DEFBASE);
 #endif
 
+
```

Comment:
Where are the additional newlines coming from? The file is auto-generated by ``Lib/ldap/constants.py`` and Makefile.

Reply:
I just ran the python script to regenerate, maybe that script changed in the meantime.